crates/nu-command/src/system/which_.rs

```diff
@@ -97,6 +99,7 @@ fn get_entries_in_nu(
 ) -> Vec<Value> {
     let mut all_entries = vec![];
 
+    // TODO: how can all_entries ever be non-empty here?
```

Comment:
just a question, will remove this TODO later!

Reply:
that is a very good question here :laughing: 

because of `let mut all_entries = vec![];`, `all_entries` is empty on the next line and thus `!all_entries.is_empty()` is always `false` regardless of the value of `skip_after_first_found` :eyes:

Reply:
so we'll want to remove this line to land this PR :wink:

Reply:
thank you and done! :)